### DISCUSSION TITLE
Get Links for Previous Current and Next Note In Folder

### DISCUSSION THREAD

**Discussion by Dercraker** [SilentVoid13/Templater#1182] [Templates Showcase]
This template gives you links to previous, current and subsequent notes to your current note.
All notes must be in the same file.
The name of the note does not matter. 

The template : [Prev|Current|Next Note In Folder](https://gist.github.com/Dercraker/83bb91f3391b61014ba4f7d1bceb7a24)

If you have any problems or ideas for improvement, please let me know.

**Comment by Gnossos** [comment]
I get a 404 error when I click the link.
